working-days/working-days.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container">
+  <form>
```

Comment:
Do not create multiple forms, create one single form which encapsulates all the fields. Also, wrap all the fields in a mat-card-content to ensure consistency in UI. See src/app/accounting-rules/create-rule/create-rule.html for reference.